diff --git a/index.js b/index.js
index f91f1ea4d0c64c41595f7b27dca4d6618145cd0d..e788b5fdb14f723441c08db1921626c2f021026e 100644
--- a/index.js
+++ b/index.js
@@ -129,7 +129,7 @@ module.exports = {
     if (!includer.options.snippetSearchPaths) {
       if (this._documentingAddonAt()) {
         // we are a standalone app, so our code is here
-        includer.options.snippetSearchPaths = ['app'];
+        includer.options.snippetSearchPaths = ['app',`${this._documentingAddonAt()}/dist`];
       } else {
         // we are inside the addon, so our code is here
         includer.options.snippetSearchPaths = ['tests/dummy/app'];
@@ -386,7 +386,7 @@ module.exports = {
     let addon;
     if (this._documentingAddonAt()) {
       addon = this.project.addons.find(
-        (a) => a.root === this._documentingAddonAt(),
+        (a) => a.root === this._documentingAddonAt() || a.root.endsWith('ember-yeti-table'),
       );
       if (!addon) {
         throw new Error(
